test/config.test.js

```diff
@@ -0,0 +1,208 @@
+'use strict'
+
+const _ = require('lodash')
+const Config = require('../src/services/config').default
+const expect = require('chai').expect
+const assert = require('chai').assert
+const logger = require('../src/common/log')
+const logHelper = require('./helpers/log')
+const env = _.cloneDeep(process.env)
+
+describe('Config', function () {
+  logHelper(logger)
+
+  describe('parseConnectorConfig', function () {
+    beforeEach(function () {
+      process.env.CONNECTOR_ACCOUNTS = JSON.stringify({
+        'usd-ledger': {
```

Comment:
Are these ledger prefixes? Shouldn't it be `"usd-ledger."` (with a trailing `.`)?

Reply:
No, the keys on the accounts map are now just some arbitrary names ("account id") that you assign. Generally, the only thing they're used for now is for referring to the account. The reason is that ILP addresses are ephemeral, so not a good idea to hard-code them in the config.

The one exception is child ledgers. The connector will by default assign the address `[own address].[child's account id]` to the child. You can override that with the `ilpAddressSegment` option in the account options.